demos/headless/headless.go

```diff
@@ -266,7 +267,7 @@ func (c *ScoreSystem) Update(entity *ecs.Entity, dt float32) {
 		c.upToDate = true
 		c.scoreLock.RUnlock()
 
-		render.Display = basicFont.Render(render.Label)
+		render.SetDrawable(basicFont.Render(render.Label))
```

Comment:
if Render is a component, then why not assign drawable to a public instance variable?